ext/serializer.c

```diff
@@ -651,6 +731,18 @@ void ddtrace_set_root_span_properties(ddtrace_span_data *span) {
         ZEND_HASH_FOREACH_END();
     }
 
+    if (getenv("DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED")
```

Comment:
```suggestion
    if (zend_hash_num_elements(get_DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED())
```
env is not necessarily set. Don't use getenv().